packages/salesforcedx-vscode-soql/src/sfdx.ts

```diff
@@ -7,6 +7,7 @@
 
 import { Connection } from '@salesforce/core';
 import { ChannelService } from '@salesforce/salesforcedx-utils-vscode/out/src/commands';
+import { WorkspaceContextUtil } from '@salesforce/salesforcedx-utils-vscode/out/src/context';
 import { DescribeSObjectResult } from 'jsforce';
 import * as vscode from 'vscode';
```

Comment:
This import can now be removed

Reply:
Fixed in `fix(soql): remove unused import` commit.